modules/gitops-configuring-namespace-scoped-argo-rollouts-installation.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/enable-support-for-namespace-scoped-argo-rollouts-installation.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-namespace-scoped-argo-rollouts-installation_{context}"]
+= Configuring namespace-scoped Argo Rollouts installation
```

Comment:
```suggestion
= Configuring a namespace-scoped Argo Rollouts installation
```
Instead of saying "Configuring a...installation", would it be accurate to simply say "Installing a namespace-scoped Argo Rollouts instance"?

Reply:
Fixed.